src/test/java/StringTest.java

```diff
@@ -0,0 +1,39 @@
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+public class StringTest {
+
+    @Test
+    void shouldSplitCorrectly() {
+        {
+            String[] actual = "1,2".split(",");
+            Assertions.assertThat(actual).containsExactly("1", "2");
+        }
+
+        {
+            String[] actual = "1".split(",");
+            Assertions.assertThat(actual).containsExactly("1");
+        }
```

Comment:
```suggestion
       assertThat("1".split(",")).containsExactly("1");
       assertThat("1,2".split(",")).containsExactly("1", "2");
```
테스트 코드도 유지보수해야할 코드인데 위와 같이 간단한 test case의 경우 위와 같이 한 라인으로 구현해 보는 것은 어떨까?